build_tools/scripts/git/check_submodule_init.py

```diff
@@ -20,13 +29,28 @@ def run():
 
   output = os.popen("git submodule status")
   submodules = output.readlines()
+  runtime_submodules = [
+      "third_party/benchmark", "third_party/cpuinfo", "third_party/flatcc",
+      "third_party/googletest", "third_party/liburing", "third_party/libyaml",
+      "third_party/musl", "third_party/spirv_cross",
+      "third_party/spirv_headers", "third_party/tracy",
+      "third_party/vulkan_headers", "third_party/vulkan_memory_allocator",
+      "third_party/webgpu-headers"
```

Comment:
Nit:I think one per line would be easier to read, less likely to cause merge conflicts, and easier to sort

Reply:
Move it to `runtime_submodules.txt`